format/Schema.fbs

```diff
@@ -357,18 +357,19 @@ enum IntervalUnit: short { YEAR_MONTH, DAY_TIME, MONTH_DAY_NANO}
 // days can differ in length during day light savings time transitions).
 // All integers in the types below are stored in the endianness indicated
 // by the schema.
+//
 // YEAR_MONTH - Indicates the number of elapsed whole months, stored as
 //   4-byte signed integers.
-// DAY_TIME - Indicates the number of elapsed days and milliseconds,
-//   stored as 2 contiguous 32-bit integers (8-bytes in total).  Support
+// DAY_TIME - Indicates the number of elapsed days and milliseconds (no leap seconds),
+//   stored as 2 contiguous 32-bit integers (8-bytes in total). Support
 //   of this IntervalUnit is not required for full arrow compatibility.
 // MONTH_DAY_NANO - A triple of the number of elapsed months, days, and nanoseconds.
-//  The values are stored contiguously in 16 byte blocks. Months and
-//  days are encoded as 32 bit integers and nanoseconds is encoded as a
-//  64 bit integer. All integers are signed. Each field is independent
-//  (e.g. there is no constraint that nanoseconds have the same sign
-//   as days or that the quantity of nanoseconds represents less
-//   than a day's worth of time).
+//  The values are stored contiguously in 16 byte blocks. Months and days are
+//  encoded as 32 bit integers and nanoseconds is encoded as a 64 bit integer.
+//  Nanoseconds does not allow for leap seconds. All integers are signed. Each
+//  field is independent (e.g. there is no constraint that nanoseconds have the
+//  same sign as days or that the quantity of nanoseconds represents less than
+//  a day's worth of time).
```

Comment:
Nit: Use dash consistently when referring to `XX-bit`.

Also, might as well state `32-bit signed integers ... 64-bit signed integer` and remove the sentence `All integers are signed`.